src/devices/src/virtio/block.rs

```diff
@@ -741,8 +663,8 @@ mod tests {
         let mut disk_image = b.disk_image.take().unwrap();
         let disk_nsectors = disk_image.seek(SeekFrom::End(0)).unwrap() / SECTOR_SIZE;
         let status = Arc::new(AtomicUsize::new(0));
-        let interrupt_evt = EventFd::new(libc::EFD_NONBLOCK).unwrap();
-        let queue_evt = EventFd::new(libc::EFD_NONBLOCK).unwrap();
+        let interrupt_evt = EventFd::new().unwrap();
+        let queue_evt = EventFd::new().unwrap();
```

Comment:
Why remove the `libc::EFD_NONBLOCK` flag on the test EventFd? In case of errors this will hang until test times-out.